examples/all-clusters-app/all-clusters-common/src/power-topology-stub.cpp

```diff
@@ -70,3 +70,16 @@ void emberAfPowerTopologyClusterInitCallback(chip::EndpointId endpointId)
         gInstance->Init();
     }
 }
+
+void emberAfPowerTopologyClusterShutdownCallback(chip::EndpointId endpointId)
+{
+    if (gInstance)
+    {
+        gInstance->Shutdown();
+        gInstance = nullptr;
+    }
+    if (gDelegate)
+    {
+        gDelegate = nullptr;
+    }
```

Comment:
```suggestion
    gDelegate = nullptr;
```

this probably applies throught: no if needed if not used and we just want to reset it.

Reply:
Sure. Done.